### ISSUE TITLE
Does NOT work on Mojave Mac

### ISSUE THREAD

**Issue by mehdi9000** [widget-/slack-black-theme#71]
![by default 2018-11-07 at 12 48 55 pm](https://user-images.githubusercontent.com/9539678/48150000-7e2cd900-e28b-11e8-9a51-0e113985d790.jpg)

By editing the " ssb-interop.js" file per the instructions the main chat window stays white in MacOS Mojave.

_Note: I used "One Dark" which uses:_ 

_"--primary: #61AFEF;
--text: #ABB2BF;
--background: #282C34;
--background-elevated: #3B4048;"_

**Below is the code I used in ssb-interop.js**

// First make sure the wrapper app is loaded
document.addEventListener("DOMContentLoaded", function() {

   // Then get its webviews
   let webviews = document.querySelectorAll(".TeamView webview");

   // Fetch our CSS in parallel ahead of time
   const cssPath = 'https://cdn.rawgit.com/widget-/slack-black-theme/master/custom.css';
   let cssPromise = fetch(cssPath).then(response => response.text());

   let customCustomCSS = 
  :root {
      /* Modify these to change your theme colors: */
      --primary: #61AFEF;
      --text: #ABB2BF;
      --background: #282C34;  
      --background-elevated: #3B4048;
   }
   

   // Insert a style tag into the wrapper view
   cssPromise.then(css => {
      let s = document.createElement('style');
      s.type = 'text/css';
      s.innerHTML = css + customCustomCSS;
      document.head.appendChild(s);
   });

   // Wait for each webview to load
   webviews.forEach(webview => {
      webview.addEventListener('ipc-message', message => {
         if (message.channel == 'didFinishLoading')
            // Finally add the CSS into the webview
            cssPromise.then(css => {
               let script = `
                     let s = document.createElement('style');
                     s.type = 'text/css';
                     s.id = 'slack-custom-css';
                     s.innerHTML = \`${css + customCustomCSS}\`;
                     document.head.appendChild(s);
                     `
               webview.executeJavaScript(script);
            })
      });
   });
});

**Comment by RyanPWalker** [comment]
Same.  Everything looks cool except the background is white which is the main thing I wanted black... 
![image](https://user-images.githubusercontent.com/25303720/48278054-ad695480-e409-11e8-9b28-d82820da4bf7.png)

**Comment by mehdi9000** [comment]
Yeah it’s broken - the other dark mode from the other author works but it doesn’t look right.

**Comment by RyanPWalker** [comment]
@GodofWar9000  I found a work around for my theme, might help you - https://github.com/widget-/slack-black-theme/issues/62#issuecomment-410256441

**Comment by oleghind** [comment]
> @GodofWar9000 I found a work around for my theme, might help you - [#62 (comment)](https://github.com/widget-/slack-black-theme/issues/62#issuecomment-410256441)

it didn't work

**Comment by aeruder** [comment]
@RyanPWalker's referenced fix worked for me on Mojave.
